src/fs-plus.coffee

```diff
@@ -287,7 +283,7 @@ fsPlus =
   # Public: Create a directory at the specified path including any missing
   # parent directories asynchronously.
   makeTree: (directoryPath, callback) ->
-    fs.exists directoryPath, (exists) ->
+    fsPlus.isDirectory directoryPath, (exists) ->
```

Comment:
> Mind adding a new spec for this and seeing what happens?

Addressed in 829fa4e7b61b838e0e077f48f62f5045502dd78e. I added `isDirectorySync` in `makeTreeSync` as well (see 076ca19de7f5ac07b7f818b9cc932f0c40e4f0db).

...I guess it's the first time when I'm writing coffeescript code. :smile: Anyway, I'm sure it's my first spec written in coffeescript. BTW, I had never drunk :coffee: in my life.